cmd/kubeadm/app/cmd/options/constant.go

```diff
@@ -117,3 +117,9 @@ const FileDiscovery = "discovery-file"
 
 // ControlPlane flag instruct kubeadm to create a new control plane instance on this node
 const ControlPlane = "experimental-control-plane"
+
+// UploadCerts flag instruct kubeadm to upload certificates to kubeadm-certs
```

Comment:
can remove the `kubeadm-certs` part since this only effects behavior and not the name of the secret